compiler/src/iree/compiler/Codegen/Common/LinkTuningSpecsPass.cpp

```diff
@@ -165,17 +145,13 @@ struct LinkTuningSpecsPass final
 FailureOr<NamedSequenceOp> linkTuningSpecs(ModuleOp module) {
   SmallVector<NamedSequenceOp> tuningSpecs;
 
+  if (failed(mlir::verify(module)))
+    return failure();
+
```

Comment:
We should verify the result of linking, not the input. It is assumed that the input would have been verifier by the parser or something else that created it.